onnx_tf/frontends/frontend_v1.py

```diff
@@ -81,25 +82,28 @@ def _conv(cls, node, d, **kwargs):
     dilations = list(
         map(lambda i: node.attr.get("dilations", [1] * (d + 2))[i],
             spatial_indices))
-    consts = kwargs["consts"]
-    output_shapes = kwargs["output_shapes"]
-    kernel_name = node.inputs[1].replace("/read", "")
-    kernel_shape = list(
-        map(lambda i: consts[kernel_name].shape[i],
-            list(range(d + 2))[d:]))
+    node_dict = kwargs["node_dict"]
+    kernel_shape = node_dict[node.inputs[1]].attr["_output_shapes"][0][:d]
     output_shape = list(
         map(lambda i: node.attr["_output_shapes"][0][i], spatial_indices))
     input_shape = list(
-        map(lambda i: output_shapes[node.inputs[0]][0][i], spatial_indices))
+        map(lambda i: node_dict[node.inputs[0]].attr["_output_shapes"][0][i],
+            spatial_indices))
     pads = cls._cal_pads(auto_pad, len(spatial_indices), input_shape,
                          output_shape, strides, kernel_shape)
-    return helper.make_node(
-        "Conv", [node.inputs[0], node.inputs[1]], [node.name],
+    unique_suffix = get_unique_suffix()
+    transpose_node = helper.make_node(
+      "Transpose", [node.inputs[1]], [node.inputs[1] + "_T_" + unique_suffix],
+      perm=[d + 1, d] + range(d))
```

Comment:
`_T_` for transpose?

Reply:
Yes. 
I'm trying to give more readability to the node in case someone tries to visualize it.